rdbms/types/operations.cpp

```diff
@@ -6,72 +6,90 @@
 #include <algorithm>
 #include <iostream>
 #include <set>
+#include<string>
 
-table operations::runion(table R1 ,table R2){
-    //conditional statment will not work tuple comparing function must be implimented
-    if (!tuples::isEqual(R1.get_attributes() , R2.get_attributes())){
-        throw "Invalid Table atteribute types can't Union the two tables";
-    }
-
-    table runion (rSchema("R1_R2_union",R1.get_attributes()));
-    for (std::set<tuples, tuplesComparator>::iterator itr1 = R1.get_rows().begin(); itr1 != R1.get_rows().end(); ++itr1) 
-    { 
-         runion.add_row(*itr1);
-    }
-    ////// adding the rows of the second table to the union
-        for (std::set<tuples, tuplesComparator>::iterator itr2 = R2.get_rows().begin(); itr2 != R2.get_rows().end(); ++itr2) 
-    {
-        runion.add_row(*itr2);
-    } 
-
-    return runion;
+table operations::runion(table R1 ,table R2)
+{
+    table t1;
+     for (auto tableRow : R1.get_rows()) {
+         for(auto tableRow1:R2.get_rows())
```

Comment:
Here also use descriptive variable names for ```tableRow``` and ```tableRow1```

Reply:
ok